csi/node.go

```diff
@@ -293,6 +293,8 @@ func (s *OsdCsiServer) NodeGetCapabilities(
 	clogger.WithContext(ctx).Debugf("csi.NodeGetCapabilities request received")
 
 	caps := []csi.NodeServiceCapability_RPC_Type{
+		// Support for Stage/Unstage of Volume on a Node.
```

Comment:
What is the purpose of adding NodeStage volume as a no-op? Are you depending on the kubelet or k8s controller manager to do something here?

Reply:
The changes needs to be seen in entireity.

This is Linux side CSI driver code changes to keep compatibility with Windows side.
As all csi drivers running in the single cluster as pxd.portworx.com will have to report same capability.
For the Linux side which traditionally has not used NodeStage this is a nop.

For the windows side, which is a new csi driver by the same 'pxd.portworx.com' it is defined as below.

a) NodeStageVolume - sets up nfs client side connections and mounts it to internal px staging site and sets up client refcounting.
b) NodePublishVolume - sets up a volume access from internal staging site to pod location, and increments ref to volume.
c) NodeUnpublishVolume - removes pod setup for volume access, decrements ref to volume.
d) NodeUnstageVolume - cleans up entire nfs client and staging path if no client references to volume exist.

Reply:
Where is the code for this new CSI driver? Is it this https://github.com/portworx/windows-csi-driver?